### DISCUSSION TITLE
battery indicator is missing

### DISCUSSION THREAD

**Discussion by balki** [starship/starship#5274] [Q&A]
Most unicode symbols are shown correctly but somehow battery indicator is missing.
![image](https://github.com/starship/starship/assets/189196/3eb02e0b-0f92-4c68-a41c-89df4fef84ac)

Is this font problem? How do I find which font is used by starship?

**Comment by mickimnet** [comment]
What version of –Starship_ are you using? _Nerd Fonts_ changed the location of some of the symbols and _Starship_ version 1.15.0 fixed that.

**Reply by balki** [reply]
❯ pacman -Qi starship  
Name            : starship
Version         : 1.15.0-1
...
Optional Deps   : ttf-font-nerd: Nerd Font Symbols preset [installed]

Would installing a different font help? I have this installed - ttf-sourcecodepro-nerd

**Reply by mickimnet** [reply]
Yes, try out a different / current _Nerd Font_. That could also be the issue.
